src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,68 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    private static final String DEFAULT_SEPARATOR_REGEX = "[,:]";
+    private static final String CUSTOM_SEPARATOR_WRAPPER_PREFIX = "//";
+    private static final String CUSTOM_SEPARATOR_WRAPPER_POSTFIX = "\n";
+    private static final String CUSTOM_SEPARATOR_INDICATOR = CUSTOM_SEPARATOR_WRAPPER_PREFIX + ".+" + CUSTOM_SEPARATOR_WRAPPER_POSTFIX;
+    private static final Pattern CUSTOM_SEPARATOR_INDICATOR_PATTERN = Pattern.compile(CUSTOM_SEPARATOR_INDICATOR);
+    private static final String BAD_EXPRESSION_NON_INTEGER = "The expression must consist only of integers separated by the specific separator.";
+    private static final String BAD_EXPRESSION_NEGATIVE_INTEGER = "The negative integer cannot be passed.";
+
+    public static int calculate(String expression) {
+        if (isNullOrEmpty(expression)) {
+            return 0;
+        }
+        String[] intStrings = split(expression);
+        return sum(intStrings);
+    }
+
+    private static boolean isNullOrEmpty(String string) {
+        return string == null || string.isEmpty();
```

Comment:
```suggestion
        return Objects.isNull(string) || string.isEmpty();
```
Objects.isNull을 사용하는것이 어떨까요 ?